src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardElements.kt

```diff
@@ -392,10 +398,21 @@ private fun getLocationDisplayPair() = buildList {
             ScoreboardData.sidebarLinesFormatted,
             ScoreboardPattern.locationPattern,
             "location"
-        ).trim() to HorizontalAlignment.LEFT)
+        ).trim() to HorizontalAlignment.LEFT
+    )
+
+    ScoreboardData.sidebarLinesFormatted.firstOrNull { ScoreboardPattern.plotPattern.matches(it) }
+        ?.let { add(it to HorizontalAlignment.LEFT) }
+}
 
-    val plotLine = ScoreboardData.sidebarLinesFormatted.firstOrNull { ScoreboardPattern.plotPattern.matches(it) }
-    if (plotLine != null) add(plotLine to HorizontalAlignment.LEFT)
+fun getPlayerAmountDisplayPair() = buildList {
+    val amount = getGroupFromPattern(TabListData.getTabList(), ScoreboardPattern.playerAmountPattern, "amount")
```

Comment:
The amount is wrong on garden and private island.

Reply:
Does it not have the (1) there?

Reply:
Missing coop support (also needs guesting support)
![image](https://github.com/hannibal002/SkyHanni/assets/24389977/195b65ca-eeed-4a87-90f7-74ed02fcfb78)
![image](https://github.com/hannibal002/SkyHanni/assets/24389977/9144e1e3-e6c9-4d0d-869e-9417bb81551f)

https://regex101.com/r/wXgVDG/1

Reply:
![image](https://github.com/hannibal002/SkyHanni/assets/45315647/db3331e9-6451-4f4c-ae50-d28bd06b2ed6)
okay yay

Reply:
ah can you send me that tablist line

Reply:
```
'          §r§b§lCoop §r§f(2)'
'§8[§r§5391§r§8] §r§bhannibal2 §r§7☢'
'§baeymii §r§7(Offline 35d+§r§7)'
'          §r§5§lGuests §r§f(0)'
'               §r§3§lInfo'
'§b§lArea: §r§7Garden'
```